### ISSUE TITLE
Consider switching to `em` unit

### ISSUE THREAD

**Issue by adamschwartz** [cmaddux/littlebox#16]
There are a number of advantages to using `em`s over `px`. See my identical issue for [icono](https://github.com/saeedalipoor/icono) (a similar library) over at https://github.com/saeedalipoor/icono/issues/16.

**Comment by cmaddux** [comment]
I've tried out `em`s for a few of the simpler icons (they're designated scalable on the demo site), but have run into trouble (maybe the rounding problems you mentioned in the icon issues) with the more complex icons. I'm not sure my use of `em`s is the most efficient, so perhaps cleaning up how I use them will help. I also didn't do a great job of maintaining uniformity in those early icons, which may be contributing to my problems in improving scalability as well. I think there's a way to do it, I just need to spend some time with it. I'll make sure to take a full look at the issue on icono, but if you have any suggestions, just let me know! Thanks again!

**Comment by adamschwartz** [comment]
Yeah, it’s definitely tricky. At [Eager](https://eager.io/) we use single element icons in our App Store.

I whipped up a little [jsFiddle](http://jsfiddle.net/adamschwartz/ra4ox9Lp/) with some of the icons we use and show how they can be rendered at various sizes. No doubt some of your more ambitions icons will make scaling tough, in part due to rounding, and in part due to the way browsers scale the rasterization of various properties (border-radius, adjoining of two-different-sized border-widths, box-shadow, etc.) But maybe this demo will help with your implementation! :smile:
